### ISSUE TITLE
onZoomStart is not called for pinch

### ISSUE THREAD

**Issue by tomeronen** [chartjs/chartjs-plugin-zoom#790]
if pinch is enabled, then zoom happens without the onZoomStart being called. I think this is a quick fix. Can i open a PR to add the onZoomStart function before the zoom call? 
```
function handlePinch(chart, state, e) {
  if (state.scale) {
    const {center, pointers} = e;
    // Hammer reports the total scaling. We need the incremental amount
    const zoomPercent = 1 / state.scale * e.scale;
    const rect = e.target.getBoundingClientRect();
    const pinch = pinchAxes(pointers[0], pointers[1]);
    const mode = state.options.zoom.mode;
    const amount = {
      x: pinch.x && directionEnabled(mode, 'x', chart) ? zoomPercent : 1,
      y: pinch.y && directionEnabled(mode, 'y', chart) ? zoomPercent : 1,
      focalPoint: {
        x: center.x - rect.left,
        y: center.y - rect.top
      }
    };

    zoom(chart, amount);

    // Keep track of overall scale
    state.scale = e.scale;
  }
}
```

**Comment by kurkle** [comment]
Fixed by #810

**Comment by romor** [comment]
When returning `false` in the onZoomStart() callback, the zoom operation should be cancelled according documentation.

However, the pinch event is not cancelled for me and the zoom is done although it should be aborted.

Thus, I think this ticket shall not be closed as it is not completely finished.
